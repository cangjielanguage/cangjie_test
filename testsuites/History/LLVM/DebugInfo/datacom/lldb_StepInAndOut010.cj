// DEPENDENCE: ../cjcmd.py ../DatacomTest.py lldb_StepInAndOut010_expected.txt ../basecase/016_debug_return.cj
// EXEC: cjc -g 016_debug_return.cj
// EXEC: python3 DatacomTest.py lldb_StepInAndOut010_expected.txt



