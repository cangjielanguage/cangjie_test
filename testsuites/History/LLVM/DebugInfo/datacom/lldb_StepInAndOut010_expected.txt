lldb,lldb 016_debug_return,016_debug_return
b,b 5,016_debug_return.cj:5
b,b 14,016_debug_return.cj:14
b,b 16,016_debug_return.cj:16
r,r,-> 5
s,s,-> 13
finish,finish,-> 16
finish,finish,-> 5
n,n,-> 6
cj_plocals,cj_plocals,varreturnexpr = 20